lib/chef/mixin/powershell_out.rb

```diff
@@ -34,7 +34,14 @@ def powershell_out(*command_args)
         script = command_args.first
         options = command_args.last.is_a?(Hash) ? command_args.last : nil
 
-        run_command_with_os_architecture(script, options)
+        result = run_command_with_os_architecture(script, options)
+
+        if result.stderr.include? "A positional parameter cannot be found that accepts argument"
+          raise "Please use single escape for special characters in powershell_out."
```

Comment:
Appended `stderr` object returned by shell_out to the raised error. `stderr` shows the specific area where the code fails. An example of raised error by `stderr` is shown below:
```
[2016-10-03T09:13:45+00:00] FATAL: Mixlib::ShellOut::ShellCommandFailed: Please use single escape for special characters in powershell_out.
 Write-Verbose : A positional parameter cannot be found that accepts argument 'String\'.
At line:1 char:1
+ Write-Verbose \Some String\ -Verbose
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Write-Verbose], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.WriteVerboseCommand
```